### ISSUE TITLE
[Fabric] Map projection of node/rel broken

### ISSUE THREAD

**Issue by ikwattro** [neo4j/neo4j#12939]
When using Fabric, map projections are broken if returned outside of a subquery call.

With the standard movies graph : 

The following works : 

```
MATCH (n:Person {name: "Keanu Reeves"})
RETURN n{.*}

╒═══════════════════════════════════╕
│"n"                                │
╞═══════════════════════════════════╡
│{"born":1964,"name":"Keanu Reeves"}│
└───────────────────────────────────┘
```

As does the following : 

```
CALL {
    MATCH (n:Person {name: "Keanu Reeves"})
    RETURN n
}
RETURN n{.*}

╒═══════════════════════════════════╕
│"n"                                │
╞═══════════════════════════════════╡
│{"born":1964,"name":"Keanu Reeves"}│
└───────────────────────────────────┘
```

But the following doesn't work

```
:use fabric

CALL {
    USE movies
    MATCH (n:Person {name: "Keanu Reeves"})
    RETURN n
}
RETURN n{.*}

╒═══╕
│"n"│
╞═══╡
│{} │
└───┘
```

**Neo4j Version:** 4.4.11
**Operating System:** Mac OS
**API:** Docker

### Expected behavior

Map projections should work with Fabric

### Actual behavior
Map projections do not work correctly with Fabric

**Comment by LinneaAndersson** [comment]
@ikwattro Thanks for reporting this issue. A fix has been merged and it will be released in the next version (4.4.14).

Again, many thanks - and let us know if you find any other issues
// Linnéa

**Comment by ikwattro** [comment]
Thanks @LinneaAndersson , looking forward to the next version.

Cheers

Chris
